src/containers/orgs/OrgsReducer.js

```diff
@@ -258,6 +261,27 @@ export default function reducer(state :Map = INITIAL_STATE, action :Object) {
       });
     }
 
+    case editOrganizationDetails.case(action.type):
+      return editOrganizationDetails.reducer(state, action, {
+        REQUEST: () => state
+          .setIn([EDIT_ORGANIZATION_DETAILS, REQUEST_STATE], RequestStates.PENDING)
+          .setIn([EDIT_ORGANIZATION_DETAILS, action.id], action),
+        SUCCESS: () => {
+          const { organizationId, title, description } = action.value;
+          const currentOrg :Organization = state.getIn([ORGS, organizationId]);
+          const updatedOrg = currentOrg.toImmutable().merge({
+            description,
+            title,
+          });
+
+          return state
+            .setIn([EDIT_ORGANIZATION_DETAILS, REQUEST_STATE], RequestStates.SUCCESS)
+            .setIn([ORGS, organizationId], new OrganizationBuilder(updatedOrg).build());
```

Comment:
shouldn't this be converted back to immutable?

Reply:
an organization is a class object and is not an immutable collection